public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/SmartAlertTypeDetector.tsx

```diff
@@ -83,98 +79,82 @@ export interface SmartAlertTypeDetectorProps {
+  // TODO figure out how to show a popover to the user to indicate _why_ it's disabled
+  const disabledOptions = canSwitch ? [] : [RuleFormType.cloudAlerting];
 
   return (
-    <div className={styles.alert}>
-      <Alert
-        severity="info"
-        title={typeTitle}
-        onRemove={canSwitch ? onClickSwitch : undefined}
-        buttonContent={`Switch to ${switchToLabel} alert rule`}
-      >
-        <Stack gap={0.5} direction="row" alignItems={'baseline'}>
-          <div className={styles.alertText}>{content?.title}</div>
-          <div className={styles.needInfo}>
-            <NeedHelpInfo
-              contentText={content?.contentText ?? ''}
-              externalLink={`https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/alert-rule-types/`}
-              linkText={`Read about alert rule types`}
-              title=" Alert rule types"
-            />
-          </div>
+    <Stack direction="column" gap={1} alignItems="flex-start">
+      <Stack direction="column" gap={0}>
+        <Text variant="h5">Rule type</Text>
+        <Stack direction="row" gap={0.5} alignItems="baseline">
+          <Text variant="bodySmall" color="secondary">
+            Select where the alert rule will be managed.
+          </Text>
+          <NeedHelpInfo
+            contentText={
+              <>
+                <Text color="primary" variant="h6">
+                  Grafana-managed alert rules
+                </Text>
+                <p>
+                  Grafana-managed alert rules allow you to create alerts that can act on data from any of our supported
+                  data sources, including having multiple data sources in the same rule. You can also add expressions to
+                  transform your data and set alert conditions. Using images in alert notifications is also supported.
+                </p>
+                <Text color="primary" variant="h6">
+                  Data source-managed alert rules
+                </Text>
+                <p>
+                  Data source-managed alert rules can be used for Grafana Mimir or Grafana Loki data sources which have
+                  been configured to support rule creation. The use of expressions or multiple queries is not supported.
+                </p>
+              </>
+            }
+            externalLink="https://grafana.com/docs/grafana/latest/alerting/fundamentals/alert-rules/alert-rule-types/"
+            linkText="Read about alert rule types"
+            title="Alert rule types"
+          />
         </Stack>
-      </Alert>
-    </div>
+      </Stack>
+      <RadioButtonGroup
+        options={options}
+        disabled={editingExistingRule}
+        disabledOptions={disabledOptions}
+        value={ruleFormType}
+        onChange={onClickSwitch}
+      />
+      {/* editing an existing rule, we just show "cannot be changed" */}
+      {editingExistingRule && <Text color="secondary">The alert rule type cannot be changed.</Text>}
```

Comment:
maybe it could be useful to add why we cannot change this type?, like `Alert rule type cannot be edited for an existing rule` ?